conanfile.py

```diff
@@ -28,12 +28,6 @@ def source(self):
         self.run("git clone --depth 1 --branch %s %s" % (branch, self.source_url))
 
     def build(self):
-        tools.replace_in_file("libexpat/expat/CMakeLists.txt", "cmake_minimum_required(VERSION 2.8.10)",
-            """cmake_minimum_required(VERSION 2.8.10)
-include(${CMAKE_BINARY_DIR}/../conanbuildinfo.cmake)
```

Comment:
Thanks @datalogics-robb for the contribution. It seems this change break the linux builds. Could you explain what's your reasoning to remove this piece of code ?

Reply:
I believe I got a little aggressive here - the 2.2.8 CMakeLists.txt requires a later version of Cmake already, so I should have just updated the patch.